index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header class="header">
+        <a href="index.html" class="logo">
+            My Logo
+        </a>
+        <nav class="icon-menu"><img src="images/icon-menu.png" alt="icon-menu" id="img-1"></nav>
+    </header>
+    <main class="main-content">
+        <section class="section-1">
+            <article class="article">
+                <h1 class="intro">
+                    Hey There.
+                    I'm Okari.
+                </h1>
+                <h1 class="job-title">
+                    I'm a software developer
```

Comment:
- [ ] Good work adding the `h1` tag as was requested by the previous reviewer, kindly use only one `h1` tag for each page as it follows best practices of HTML. In this case would be great to leave only your `h1` with class `article`